README.md

```diff
@@ -12,4 +12,54 @@ To convert a theme use this command:
 ./bin/console prestashop:mail:convert-mjml modern_mjml {twig_theme_path}
 ```
 
+On Windows use
+
+```
+php /bin/console prestashop:mail:convert-mjml modern_mjml {twig_theme_path}
```

Comment:
The difference with above is the `php /` ?

Reply:
Yes, but `/` is a typo, sorry.